ivy/functional/frontends/jax/random.py

```diff
@@ -171,3 +172,19 @@ def t(key, df, shape=(), dtype="float64"):
 def randint(key, shape, minval, maxval, dtype="int64"):
     seed = _get_seed(key)
     return ivy.randint(minval, maxval, shape=shape, dtype=dtype, seed=seed)
+
+
```

Comment:
Kindly add the `@to_ivy_arrays_and_back` and `@handle_jax_dtype` decorators to the function here